src/meshes/p4est_mesh.jl

```diff
@@ -191,6 +191,10 @@ function P4estMesh(trees_per_dimension; polydeg,
                    p4est_partition_allow_for_coarsening = true)
     @assert ((coordinates_min === nothing)===(coordinates_max === nothing)) "Either both or none of coordinates_min and coordinates_max must be specified"
 
+    if coordinates_min !== nothing && coordinates_max !== nothing
+        coordinate_min_max_check(coordinates_min, coordinates_max)
+    end
```

Comment:
```suggestion
    coordinate_min_max_check(coordinates_min, coordinates_max)
```
If you accept my proposal from above.